package/requirements.txt

```diff
@@ -1,4 +1,4 @@
 semver~=2.10 # Needs to be at least 2.10.0 to get VersionInfo.match
-Flask>=1.0, <2.0
+Flask>=2.0
```

Comment:
Is this change related to / required by this PR?

Reply:
Just saw the commit history and it looks like this has been updated yesterday from lim - perhaps this should be introduced in a seperate PR?

Reply:
Yeah this was a temporary fix by Lim, I think this is resolved in the main branch now though which I've merged here.